src/common/hexchat.c

```diff
@@ -883,9 +883,9 @@ xchat_init (void)
 		"NAME ENDSUB\n"				"CMD \n\n",
 
 		_("_Open Dialog Window"), "gtk-go-up",
-		_("_Send a File"), "gtk-floppy",
+		_("_Send a File..."), "gtk-floppy",
```

Comment:
VS 2017 displays ellipsis characters just fine when using UTF-8, hopefully the compiler will accept them too. In the Windows UI font (Segoe UI), the character looks exactly the same as just three periods, should probably check that it doesn't look squashed on other platforms.